ChangeLog.txt

```diff
@@ -1,8 +1,31 @@
 Next v0.1.X
 -----------
 
+Cultures:
+
+- Added namelist for Vandal culture
```

Comment:
What source did you use?

Reply:
> What source did you use?

Mostly these ones:
https://www.academia.edu/1516556/THE_LANGUAGE_AND_NAMES_OF_THE_VANDALS
https://www.academia.edu/691311/Tracing_the_Language_of_the_Vandals

Reply:
Ok on the sources, but the number of female names is too low. Is there a pool that we can complement with from other similar Germanic cultures?